### DISCUSSION TITLE
Add support for multiple languages UI and subtitles

### DISCUSSION THREAD

**Discussion by renansimoes** [LostArtefacts/TRX#885] [Ideas]
It would be nice if we could translate the game UI and also add subtitles for cutscenes and FMV.

**Comment by aredfan** [comment]
I agree, it would be nice 😄 

Afaik these are the relevant issues for this discussion.  It's worth noting that T1M is still being developed so it could be a while before these features happen.

- #382 
- #386 
- #466 
- #626 
- #715 
- #1021 
- #1097

**Reply by Richard-L** [reply]
Good collection!

**Reply by walkawayy** [reply]
I renamed a bunch of these issues to hopefully be clearer. Because they were named so similarly it was confusing. I think #626 can be closed since it seems like a duplicate of what unicode would provide?

**Reply by aredfan** [reply]
Yeah it was confusing since some of the issues have been re-worked recently, so the titles needed clarifying.  

If 626 is closed then I'll be happy to remove it from the list too. 👍

**Comment by Kappa971** [comment]
The only thing missing is a subtitle management system for in-game and pre-rendered cutscenes.

**Comment by aredfan** [comment]
The multiple languages UI had been implemented for some time, and the request for subtitles is captured in various issues.

I'll close this discussion due to inactivity but it can be reopened if needs be.
